### ISSUE TITLE
ext-curly: skip LabeledStatement

### ISSUE THREAD

**Issue by ajafff** [ajafff/tslint-consistent-codestyle#51]
```ts
if (foo) { // shouldn't complain about braces here
  label: while (true) {
  }
}
```
